jsx/webpack.config.js

```diff
@@ -28,7 +28,7 @@ module.exports = {
   output: {
     publicPath: "/",
     filename: "admin-react.js",
-    path: path.resolve(__dirname, "build"),
+    path: path.resolve(__dirname, "../share/jupyterhub/static/js/"),
```

Comment:
This is what removes the need for `npm run place`. Set the destination to the actual destination.